Sources/EpoxyBars/BarModel/AnyBarModel.swift

```diff
@@ -60,3 +61,51 @@ extension AnyBarModel: Diffable {
     return model.isDiffableItemEqual(to: otherDiffableItem)
   }
 }
+
+// MARK: CallbackContextEpoxyModeled
+
+extension AnyBarModel: CallbackContextEpoxyModeled {
+	public typealias CallbackContext = Void
+}
+
+// MARK: WillDisplayProviding
+
+extension AnyBarModel: WillDisplayProviding {}
+
+// MARK: InternalBarModeling
+
+extension AnyBarModel: InternalBarModeling {
+  func makeConfiguredView(traitCollection: UITraitCollection) -> UIView {
+    UIView()
```

Comment:
should I have some preconditionFailures for these unimplemented functions? These are not intended to be called other than `willDisplay`

Reply:
Yes, let's use `EpoxyLogger.shared.assertionFailure(…)` with a message detailing that these are currently unimplemented

Reply:
Will do